vj4/ui/templates/contest_scoreboard.html

```diff
@@ -22,7 +22,7 @@
           {%- for column in rows[0] -%}
             <th class="col--{{ column['type'] }}">
             {% if column['type'] == 'problem_detail' %}
-              <a href="{{ reverse_url('contest_detail_problem', tid=tdoc['_id'], pid=column['raw']['_id']) }}"  data-tooltip="{{ column['raw']['title'] }}">{{ column['value'] }}</a>
+              <a href="{{ reverse_url('contest_detail_problem', tid=tdoc['_id'], pid=column['raw']['doc_id']) }}"  data-tooltip="{{ column['raw']['title'] }}">{{ column['value'] }}</a>
```

Comment:
this is a bug fix

Reply:
Please fix `tid=tdoc['_id']` ^\_^ T\_T

Reply:
should be fixed in another standalone PR later.